docs/contributing.rst

```diff
@@ -36,7 +36,7 @@ We use some external dependencies, multiple interpreters and code coverage
 analysis while running test suite. Our ``noxfile.py`` handles much of this for
 you::
 
-  $ nox --reuse-existing-virtualenvs --sessions test-3.8 test-3.9
+  $ nox --reuse-existing-virtualenvs --sessions test-3.8 test-3.9 test-pypy3.8 test-pypy3.10
```

Comment:
Believe it or not, we make sure the test below matches the command run. Since you're changing that file:

 * Please remove the output in lines 41-50 so that we don't have to update it anymore.
 * Let's also use more recent versions, such as 3.10 for PyPy and 3.12 for CPython, to avoid having to update the file for the Python 3.8 and 3.9 removal. This applies to your note about `PATH` too.